src/components/sort-control/sort-control.test.jsx

```diff
@@ -0,0 +1,82 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import SortControl from './sort-control';
+
+describe('SortControl Component', () => {
+  const mockOptions = [
+    { id: 1, value: 'Release Date' },
+    { id: 2, value: 'Title' },
+  ];
+  const mockOnSelect = jest.fn();
```

Comment:
It's usually better to define mocks locally in each test. One thing you could do to make it reusable is to create a setup function